modules/dr-restoring-cluster-state.adoc

```diff
@@ -296,7 +296,12 @@ Perform the following step only if you are using `OVNKubernetes` network plugin.
 ====
 +
 . Restart the Open Virtual Network (OVN) Kubernetes pods on all the hosts.
-
++
+[NOTE]
+====
+Validating webhooks may reject pods. If you add any additional validating webhooks and those webhooks reject pods, then you must disable the webhooks while restoring the cluster state. After restoring the cluster state successfully, you can enable the webhooks. If the` failurePolicy` is set to `Fail`, the process of restoring the cluster state fails. To avoid this failure, you can temporarily	set the `failurePolicy` to `Ignore`.
```

Comment:
```suggestion
Validating and mutating admission webhooks may reject pods. If you add any additional webhooks and the `failurePolicy` is set to `Fail` in those webhooks, then they may reject pods. As a result the restoration process may fail. This can be avoided in two ways. One way is to disable (save and delete) those webhooks while restoring the cluster state. After the cluster state is restored successfully, the webhooks can be enabled (recreate) again. The other way is to temporarily set the `failurePolicy` to `Ignore` if it is set to `Fail` in the webhooks while restoring the cluster state. After the cluster state is restored successfully,  `failurePolicy`  can again be set to `Fail`.
```

Reply:
Addressed